test/vaadin-list-mixin.html

```diff
@@ -479,6 +479,86 @@
           expect(list._scrollerElement.scrollTop).to.be.greaterThan(0);
         });
       });
+
+      describe('Multiple', () => {
+        const ctrlKeyClick = new MouseEvent('click', {
+          bubbles: true,
+          cancelable: true,
+          view: window,
+          ctrlKey: true
+        });
+
+        const shiftKeyClick = new MouseEvent('click', {
+          bubbles: true,
+          cancelable: true,
+          view: window,
+          shiftKey: true
+        });
+
+        it('should clear selected when multiple=true', () => {
```

Comment:
Did you see this kind of behavior in some other component, like the native <select>? To me, it seems unnecessary to clear the selection.

Going the other direction, setting multiple=false could retain the first selected item in the set and not clear the whole selection.